statescript/executor.go

```diff
@@ -243,9 +243,9 @@ func execute(name string, timeout time.Duration) error {
 
 	if len(bts) > 0 {
 		if len(bts) > 10*1024 {
-			log.Errorf("Collected standard-error while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
+			log.Infof("Collected output (stderr) while running script %s (Truncated to 10KB)\n%s\n---------- end of script output", name, bts[:10*1024])
```

Comment:
I think this is originally my handy-work actually, hah.

Anywho. Seems reasonable.

:+1:  from me

Reply:
Could you submit this as a separate PR? This commit is a very obvious improvement which should be accepted immediately, whereas the other code may require more work/discussion. Good to get it out of the way.